docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -10,16 +10,54 @@ device-mqtt features using an MQTT-broker.
 
 ![MQTT Overview](MQTT_Example_Overview.png)
 
-## Prepare the Custom Device Configuration 
+## Enabling Multi-Level Topics
+Multi-Level Topics move metadata (i.e. device name, command name,... etc) from the payload into the MQTT topics. 
+
+To use the optional setting for MQTT device services with multi-level
+topics, make the following changes in the device service configuration files:
+
+1. Modify these lines in `configuration.toml`:
+    ``` toml
+    # Comment out/remove when using multi-level topics
+    #IncomingTopic = "DataTopic"
+    #ResponseTopic = "ResponseTopic"
+    #UseTopicLevels = false
+    
+    # Uncomment to use multi-level topics
+    IncomingTopic = "incoming/data/#"
+    ResponseTopic = "command/response/#"
+    UseTopicLevels = true
```

Comment:
These need to be done via Env Overrides in the compose file since we are not volume mounting the modified TOML file.:

```
      MQTTBROKERINFO_INCOMINGTOPIC: "incoming/data/#"
      MQTTBROKERINFO_RESPONSETOPIC: "command/response/#"
      MQTTBROKERINFO_USETOPICLEVELS: "true"
```

Reply:
done